src/main/java/com/julienvey/trello/Trello.java

```diff
@@ -3,22 +3,7 @@
 import java.io.File;
 import java.util.List;
 
-import com.julienvey.trello.domain.Action;
-import com.julienvey.trello.domain.AddMemberToBoardResult;
-import com.julienvey.trello.domain.Argument;
-import com.julienvey.trello.domain.Attachment;
-import com.julienvey.trello.domain.Board;
-import com.julienvey.trello.domain.Card;
-import com.julienvey.trello.domain.CardWithActions;
-import com.julienvey.trello.domain.CheckItem;
-import com.julienvey.trello.domain.CheckList;
-import com.julienvey.trello.domain.Entity;
-import com.julienvey.trello.domain.Label;
-import com.julienvey.trello.domain.Member;
-import com.julienvey.trello.domain.MemberType;
-import com.julienvey.trello.domain.MyPrefs;
-import com.julienvey.trello.domain.Organization;
-import com.julienvey.trello.domain.TList;
+import com.julienvey.trello.domain.*;
```

Comment:
let's keep the original imports. wildcard imports may bring hidden build failures.